plugins/freshdesk/plugin.spec.yaml

```
@@ -1030,4 +1198,33 @@ actions:
         description: List of tickets
         type: '[]ticket'
         required: true
-        example: '[]'
\ No newline at end of file
+        example: '[]'
+  filterTickets:
+    title: Filter Tickets
+    description: Use custom ticket fields that you have created in your account to filter through the tickets and get a list of tickets matching the specified ticket fields
+    input:
+      query:
+        title: Query
+        description: Case sensitive query to filter against, supported fields are agent_id, group_id, priority, status, tag, type, due_by, fr_due_by, created_at, updated_at. Max 512 characters
+        type: string
+        required: false
+        example: "(ticket_field:integer OR ticket_field:'string') AND ticket_field:boolean"
+      page:
+        title: Page
+        description: To scroll through the pages add page parameter to the url. The page number starts with 1 and should not exceed 10
+        type: integer
+        required: false
+        example: 2
```

Should we add a default here? Would make sense to have it default to the first page
```suggestion
        title: Page
        description: To scroll through the pages add page parameter to the url. The page number starts with 1 and should not exceed 10
        type: integer
        required: false
        default: 1
        example: 2
```

---

No need to default as if this is empty the first page is returned.